src/pages/Animeworld/main.ts

```diff
@@ -0,0 +1,48 @@
+import { pageInterface } from '../pageInterface';
+
+export const Animeworld: pageInterface = {
+  name: 'Animeworld',
+  domain: 'https://www.animeworld.tv',
+  languages: ['Italian'],
+  type: 'anime',
+  isSyncPage(url) {
+    if (url.split('/')[4] !== undefined && url.split('/')[4].length) {
+      return true;
+    }
+    return false;
+  },
+  sync: {
+    getTitle(url) {
+      return j.$('#anime-title').text().trim();
+    },
+    getIdentifier(url) {
+      return utils.urlPart(url, 4);
+    },
+    getOverviewUrl(url) {
+      return `${Animeworld.domain}/${Animeworld.sync.getIdentifier(url)}`;
+    },
+    getEpisode(url) {
+      const episode = Number(
+        j
+          .$('div.widget-body>div.server.active>ul.episodes.range.active>li.episode>a.active')
```

Comment:
This selector will break with every small page html change

Reply:
I'll change it 👍

Reply:
I hope now it could work better